File: Python/Lab1/Variant5.py
from math import *
a = float(input("Введите значение a: "))
b = float(input("Введите значение b: "))
t = float(input("Введите значение t: "))
y = exp(-b*t)*sin(a*t+b)-sqrt(fabs(b*t+a))
print("\nЗначение y =", y)

File: Pascal/Lab1/Variant5.pas
Program lab1;
Uses Crt;
Var a,b,t,Formula: real;
Begin
  Write('Введите первое число: '); Readln(a);
  Write('Введите второе число: '); Readln(b);
  Write('Введите третье число: '); Readln(t);
  ClrScr;
  Formula:=exp(-b*t)*sin(a*t+b)-sqrt(abs(b*t+a));
  Writeln('Вариант 5.  ',Formula);
End.